### ISSUE TITLE
v2: Wind speed color inverted on white background

### ISSUE THREAD

**Issue by egmontkob** [chubin/wttr.in#371]
Hi,

On white background, the wind speed color is counterintuitive (before it becomes red). Prominently visible dark gray denotes small wind, less prominent light gray numbers denote stronger wind.

![wttr-wind-speed](https://user-images.githubusercontent.com/5750745/68203726-bf52e880-ffc6-11e9-9438-7e0529633027.png)

Could you maybe consider an option to flip them?

**Comment by chubin** [comment]
Yes, it is possible. Does the `I` option works for you? (for the main version) Should it be the same here? (v2)

**Comment by egmontkob** [comment]
Am I doing it right: `curl 'wttr.in/Budapest?I'` ?

I get somewhat different colors: darker clouds and Sun, and darker colors for the temperature. The Sun's color is IMHO too dark, for the rest the result is easier to read.

I can't see such a grayscale thing here though that could help me confirm that this is what I'm exactly looking for. But I trust you that it makes sense to do this for v2, too :)

(Side note: this `I` isn't mentioned on `:help`.)

**Comment by chubin** [comment]
A new idea, @egmontkob what do you think about it?

Display speed of wind in several lines instead of one,
where the height of the plot is depending on the wind speed.
Maybe even the Beaufort number for this windspeed divided
by two (or three), so that the fastest wind with the hurricane speed,
will be displayed with a 6 lines block (or 4 lines) and light breeze with one character.

Something like this:

```
│          ↙  ←  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙              →  →  → │
│    ↙  ↙  ↙  ←  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙           →  →  →  → │
│ ↙  ↙  ↙  ↙  ←  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↖  ↓  ↘  →  →  →  → │
│ ↙  ↙  ↙  ↙  ←  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↙  ↖  ↓  ↘  →  →  →  → │
│ 14 17 19 23 30 32 35 34 33 32 33 30 31 31 29 24 21 19 18 15 18 22 24 25│

```

In my opinion, color alone is not enough to represent the windspeed,
and it is better to use something with variable size for that.

What do you think?

**Comment by egmontkob** [comment]
I'm not sure. I think I'd prefer the colored, single-line variant.
